apps/email/js/cards/message_list.js

```diff
@@ -681,7 +681,13 @@ return [
       // date
       var dateTime = message.date.valueOf();
       dateNode.dataset.time = dateTime;
-      dateNode.textContent = dateTime ? date.prettyDate(message.date) : '';
+      if (dateTime) {
+        date.relativeDate(message.date).then(string => {
+          dateNode.textContent = string;
+        });
+      } else {
+        dateNode.textContent = '';
+      }
       // subject
```

Comment:
I added an experimental node localization to RelativeDate. Let's see how it will work for you :)